src/commands/violations/revokeallname.js

```diff
@@ -30,7 +30,7 @@ class RevokeAllname extends Command {
 		if (!config.apikey) return message.reply("No API key set")
 		const offenseRaw = await fetch(`${this.client.config.apiurl}/offenses/getcommunity?playername=${strictUriEncode(playername)}&communityid=${strictUriEncode(config.communityid)}`)
 		const offense = await offenseRaw.json()
-		if (offense === null)
+		if (!offense || offense === {})
```

Comment:
`offense === {}` will always return false so does nothing useful.

Reply:
Yeah, that was because of `express-mung` not sending `null` and just sending 204: No Content. fixed it in [bb07fed](https://github.com/oof2win2/fagc-backend/commit/bb07fed738ddca1328fa1330f6bde0687b104fea) by just not using it, and since it hasn't been updated for the past few years, I just copied the code from source and removed the No Content part.
Only issue is that instead of sending 204: No Content, it sends a 404: Not Found, but that is true too, as the violation/offense etc could not be found